README.md

```diff
@@ -384,6 +384,24 @@ on the wiki.
 
 Bats is [SemVer compliant](https://semver.org/).
 
+*0.5.0* (June xx, 2018)
```

Comment:
This should be `1.0.0` now. Let's go ahead and put `June 8, 2018` in there, too. (If we slip, I can tweak it when it's time to tag and release per #16.)